### ISSUE TITLE
Stack update/create check appears to not be working

### ISSUE THREAD

**Issue by k8soneill** [UKCloud/openshift-heat#77]
Recent ld scale up showed the bastion pulling new code on an update which it should not do

**Comment by stevemul** [comment]
@koneill94 did you address this in a pull request already? I have a recollection that we dealt with it?

**Comment by k8soneill** [comment]
@stevemul Nope, as far as I'm aware this is still an issue. I'll check for a ticket and make sure it gets pulled into the next sprint.
